apps/address-service/domains/address/schema/Address.js

```diff
@@ -34,6 +36,28 @@ const Address = new GQLListSchema('Address', {
             isRequired: true,
         },
 
+        overrides: {
+            schemaDoc: `The list of overrides for address ${OVERRIDING_ROOT} field`,
+            adminDoc: `This is a JSON object that must look like {"field":"value"}. Will use to override ${OVERRIDING_ROOT}`,
+            type: Json,
+            isRequired: false,
+            access: {
+                create: access.canManageOverrides,
+                update: access.canManageOverrides,
+            },
+            hooks: {
+                validateInput: async (data) => {
+                    const { resolvedData, addFieldValidationError, existingItem, fieldPath } = data
+                    Object.entries(get(resolvedData, fieldPath, {}) || {}).forEach(([path, value]) => {
```

Comment:
What if I'll pass `[]` here? Guess object.enties will be broken

Reply:
Looks like it's no problem
https://user-images.githubusercontent.com/25384290/219364363-b5dbe5f7-fdf6-465d-ab76-b3c1c7819fd4.mov

Reply:
Anyway, just improved validation and tests: https://github.com/open-condo-software/condo/pull/2913/commits/7021d91d9a495d818953eb9ce2c9c56bd966b6b2